cluster_generator/bin/resources/particle_fields.yaml

```diff
@@ -0,0 +1,32 @@
+# Particle Field defaults for use in the particles.py module.
```

Comment:
It's not clear to me that we need this file. I like having this info at the top of `particles.py` so that it's easily accessible if you're editing in an IDE, so I'd prefer we keep it there.

Reply:
👍🏻 Removed the ``particle_fields.yaml`` file added back the original dictionaries. The ``/bin/resources/`` file is still there for later use in future pull requests.